### ISSUE TITLE
UI Filtering: Selection

### ISSUE THREAD

**Issue by dannycpham** [hackforla/lucky-parking#515]
### User Story

As a user, I want to be able to filter data down to a more granular level to visually see citation data for a region and time frame.

### Description

When a user selects an input field, the user needs to see a different UI matching with the corresponding input type.

A user must see the appropriate header matching the data type that the user had selected; Neighborhood Council if they had picked a neighborhood council data type and Zip Code if they had picked a Zip Code. 

On this new UI, there should be two main dropdowns: Citation Type and Date Range.

**Citation Type Dropdown Options:**
- Total # Citations
- Citations by Population (Density)
- Violation Type
- Total $ Fine Amount
- Fine $ Amount by Population (Density)
- Day of the Week

**Date Range Dropdown Options**

- Past 1 Year
- Year to Date
- Past 3 Months
- Past 1 Month
- All Time

As part of this new UI, there should also be a From and To Calendar link options. 

Please leave an area below this the date range drop down and calendar from and to blank for future graphs. Graphs are out of scope for this ticket.

### Acceptance Criteria

1. If a user selects neighborhood council as the input field, then the user would see a new UI appear with Citation type, Date Dropdowns, Date Range, and a graphical display of data.
2. If a user selects neighborhood council as the input field, then the user would see Neighborhood Council as the region type in the header. If a user selects zip code as the input field, then the user would see Zip Code as the region type in the header.
3. If a user selects on the dropdown, then the user would see the full list of dropdown options for both citation types and date range.

### Design References

[Figma](https://www.figma.com/file/R3N7mvDtlFbFgQm4OXcMih/Lucky-Parking?type=design&node-id=731-243&mode=design&t=O6uPiQpCRAKlc3Yd-0)

**Comment by glenflorendo** [comment]
Blocked. We need to build a dropdown/picklist component first.

**Comment by glenflorendo** [comment]
Unblocked.

**Comment by glenflorendo** [comment]
@gibsonliketheguitar There are stubs for this. Please refactor to use the picklist and other relevant components from our UI library?

**Comment by gibsonliketheguitar** [comment]
<img src="https://media4.giphy.com/media/GCvktC0KFy9l6/giphy.gif"/> @glenflorendo I'll take a look at it. 
Btw should i refactor the NIT comments from the previous PR?

**Comment by glenflorendo** [comment]
> <img src="https://media4.giphy.com/media/GCvktC0KFy9l6/giphy.gif"/> @glenflorendo I'll take a look at it. 
> 
> Btw should i refactor the NIT comments from the previous PR?

Yeah, let's do it, if it won't bloat the PR. Thanks!

**Comment by glenflorendo** [comment]
@gibsonliketheguitar, what's your status on this? Did you make any progress? If not, let's give this task to @bennyv8 .
